llmproxy/tests/unit_tests/models/test_vertexai.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from llmproxy.models.vertexai import VertexAI
+from dotenv import load_dotenv
+
+load_dotenv(".env.test")
+project_id = os.getenv("GOOGLE_PROJECT_ID")
+
+def test_invalid_project_id() -> None:
+    vertexai = VertexAI(project_id="invalid id")
+    response = vertexai.get_completion()
+    assert "Permission denied" in response.message
+
+def test_invalid_location() -> None:
+    vertexai = VertexAI(project_id=project_id, location="test")
+    response = vertexai.get_completion()
+    assert "Unsupported region" in response.message
+
+def test_unsupported_model() -> None:
```

Comment:
Would be best if you did Arrange, Act, Assert pattern explicitly: Check Mistral unit tests as an example

Reply:
added